konan_sdk/sdk.py

```diff
@@ -19,88 +18,30 @@ def __init__(self, auth_url="https://auth.konan.ai", api_url="https://api.konan.
         self.password = ''
 
         self.user: Optional[KonanUser] = None
+        self.auth: Optional[KonanAuth] = None
 
         if not verbose:
             logger.remove()
             logger.add(sys.stderr, level="INFO")
 
     def login(self, email: str, password: str) -> KonanUser:
```

Comment:
No need to return the `KonanUser` object, since the other `KonanSDK` methods internally use the `self.user` or `self.auth.user` fields and its not needed outside of a `KonanSDK` object.